aptos-move/framework/aptos-framework/sources/nonce_validation.move

```diff
@@ -0,0 +1,236 @@
+module aptos_framework::nonce_validation {
+    use aptos_std::table::{Self, Table};
+    use aptos_std::timestamp;
+    use aptos_std::big_ordered_map::{Self, BigOrderedMap};
+    use aptos_std::aptos_hash::sip_hash_from_value;
+    use aptos_std::error;
+    friend aptos_framework::genesis;
+    friend aptos_framework::transaction_validation;
+
+    const NUM_BUCKETS: u64 = 50000;
+
+    // Max expiration time for orderless txns.
+    // We will advertise to the community that max expiration time for orderless txns is 60 seconds.
+    // Adding a 5 second slack here as the client's time and the blockchain's time may drift.
+    const MAX_EXPIRATION_TIME: u64 = 65;
```

Comment:
nit: I think specifying unites in the constant name can help, also add a note here that if this is changed, the same constant needs to be changed in `transaction_validation.move`?

Reply:
Removed the constant.